### PR TITLE
fix(screencast): correctly process videos with 1 frame

### PR DESCRIPTION
**Pull request by yury-s**

### CODE REVIEW

File: src/server/chromium/videoRecorder.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@
  */
 
 import { ChildProcess } from 'child_process';
+import { time } from 'console';
```
Comment (pavelfeldman) [inline src/server/chromium/videoRecorder.ts]: stray import

File: src/server/chromium/videoRecorder.ts [hunk 2]
```diff
@@ -124,8 +126,8 @@ export class VideoRecorder {
 
     if (this._lastWriteTimestamp) {
       const durationSec = (Date.now() - this._lastWriteTimestamp) / 1000;
-      if (durationSec > 1 / fps)
-        this.writeFrame(this._lastFrameBuffer!, this._lastFrameTimestamp + durationSec);
+      if (!this._lastWritePromise || durationSec > 1 / fps)
+        this._flushLastFrame(durationSec).catch(e => this._progress.log('Error while writing frame: ' + e));
```
Comment (pavelfeldman) [inline src/server/chromium/videoRecorder.ts:129]: Shouldn't you await it?

Comment (yury-s) [reply src/server/chromium/videoRecorder.ts:129]: No, we chain them and wait for the tail in stop().

File: src/server/chromium/videoRecorder.ts (hunk 1 above)
Comment (yury-s) [reply src/server/chromium/videoRecorder.ts]: Fixed.

File: src/server/chromium/videoRecorder.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/server/chromium/videoRecorder.ts:129]: Can we chain them explicitly?

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**